kartothek/io/iter.py

```diff
@@ -200,21 +200,23 @@ def update_dataset_from_dataframes__iter(
 
     Parameters
     ----------
-    df_generator: Iterable[Union[pd.DataFrame, Dict[str, pd.DataFrame]]]
-        The dataframe(s) to be stored.
 
     Returns
     -------
     The dataset metadata object (:class:`~kartothek.core.dataset.DatasetMetadata`).
     """
+    if load_dynamic_metadata is True:
```

Comment:
should be `is not None` since the entire parameter should've been removed in `3.0`

Reply:
Aah, stupid mistake. Thanks.

I would recommend it being `is not True` as the default value is `True`